test/test_aqua.jl

```diff
@@ -1,4 +1,9 @@
 using DynamicQuantities
 using Aqua
 
-Aqua.test_all(DynamicQuantities; project_toml_formatting=false)
+Aqua.test_all(
+    DynamicQuantities;
+    project_toml_formatting=false,
+    # Requires is only loaded on older Julia versions that do not support extensions
+    stale_deps=(ignore=isdefined(Base, :get_extension) ? [:Requires] : Symbol[],),
```

Comment:
Just wondering if this is still needed with the `@require_extensions` part?

Reply:
No, it's not. I thought I had removed it but I guess I messed up the git rebase.